incubator/airflow/templates/statefulsets-workers.yaml

```diff
@@ -0,0 +1,128 @@
+      {{- if .Values.dags.init_container }}
+      initContainers:
+        - name: git-clone
+          image: alpine/git # Any image with git will do
+          command:
+          - /usr/local/git/git-clone.sh
+          args:
+          - "{{ .Values.dags.git.url }}"
+          - "{{ .Values.dags.git.ref }}"
+          - "/dags"
+          volumeMounts:
+          - name: git-clone
+            mountPath: /usr/local/git
+          - name: dags-data
+            mountPath: /dags
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}-worker
+          imagePullPolicy: {{ .Values.airflow.image.pull_policy }}
+          image: "{{ .Values.airflow.image.repository }}:{{ .Values.airflow.image.tag }}"
+          envFrom:
+          - configMapRef:
+              name: "{{ template "airflow.fullname" . }}-env"
+          env:
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "airflow.fullname" . }}
+                  key: postgresUser
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "airflow.fullname" . }}
+                  key: postgresPassword
+            - name: REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "airflow.fullname" . }}
+                  key: redisPassword
+          volumeMounts:
+          {{- if .Values.persistence.enabled }}
+            - name: dags-data
+              mountPath: {{ .Values.dags.path }}
+          {{- else if .Values.dags.init_container }}
+            - name: dags-data
+              mountPath: {{ .Values.dags.path }}
+            - name: scripts
+              mountPath: /usr/local/scripts
+          {{- end }}
+          args:
+            - "bash"
+            - "-c"
+          {{- if and ( .Values.dags.init_container ) ( .Values.dags.init_container.install_requirements ) }}
+            - >
+              echo 'waiting 60s...' &&
+              sleep 60 &&
+              echo 'installing requirements...' &&
+              /usr/local/scripts/install-requirements.sh &&
+              echo 'executing worker...' &&
+              airflow worker
+          {{- else }}
+            - >
+              echo 'waiting 60s...' &&
+              sleep 60 &&
+              echo 'executing worker...' &&
+              airflow worker"
+          {{- end }}
+          ports:
+            - name: wlog
+              containerPort: 8793
+              protocol: TCP
+          resources:
+{{ toYaml .Values.workers.resources | indent 12 }}
+      volumes:
+        - name: dags-data
+        {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+              claimName: {{ .Values.persistence.existingClaim | default (include "airflow.fullname" .) }}
+        {{- else }}
+          emptyDir: {}
+        {{- end }}
+        {{- if .Values.dags.init_container }}
```

Comment:
`{{- if .Values.dags.init_container }}` should be `{{- if .Values.dags.init_container.enabled }}`, i guess.